### PR TITLE
NEPT-1171: Implement the components in WYSIWYG via custom stylesheet

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

Add a CSS class to the WYSIWYG field container (field display). 
This class is a namespace allowing identify WYSIWYG elements that must be displayed with the europa corporate style.

### Change log

- Added: In the europa theme, add the logic to inject the namespace on the field value container.

### Commands

```sh
drush cc all

```

### CODE REVIEW

File: templates/field/field.component.inc [hunk 1]
```diff
@@ -9,36 +9,44 @@
...
     case 'field_core_social_network_links':
-      $variables['element']['before'] = t('Follow the latest progress and get involved.');
-      $variables['element']['after'] = l(t('Other social networks'), variable_get('dt_core_other_social_networks_link', 'http://europa.eu/contact/social-networks/index_en.htm'));
+      $element['before'] = t('Follow the latest progress and get involved.');
+      $element['after'] = l(t('Other social networks'), variable_get('dt_core_other_social_networks_link', 'http://europa.eu/contact/social-networks/index_en.htm'));
       break;
   }
-  if ($variables['element']['#field_type'] <> 'ds') {
-    // Get more field information.
-    $field = field_info_field($variables['element']['#field_name']);
```
Comment (voidtek) [inline templates/field/field.component.inc]: Why do you remove the function field_info_field() ?

**Comment by imanoleguskiza** [comment]
👍

**Comment by imanoleguskiza** [comment]
👍

**Comment by Fefaine** [comment]
👍

**Pull request merged**